EmpireLegends/Cards.cpp

```diff
@@ -179,7 +180,8 @@ Card* Hand::exchange(int rowPosition, Player* player)
 			player->payCoin(cardCost);
 		}
 	}
-	else
+	//Only printing for human players
+	else if(nonBot== std::string::npos)
```

Comment:
following my previous comment, this line would then become `else if(!isBot)` and could intuitively be read as "else if you aren't a bot, execute the following code"